packages/core/src/api/meta/initialize.ts

```diff
@@ -65,6 +66,9 @@ export function initializeMetaAPI(
 
   const getView: MetaAPI['getView'] = () => metas.value.view;
 
+  const getBuildId: MetaAPI['getBuildId'] = (appName) =>
```

Comment:
Suggestion:
`appName` is optional. Let's attach random ID if `appName` is missing.